### PR TITLE
NN-2931 View appointments updates

### PR DESCRIPTION
**Pull request by jsrobertson**
Add prisoner cell location column and update some content.
image

### CODE REVIEW

File: backend/routes/appointments/viewAppointmentsRouter.js [hunk 1]
```diff
@@ -88,13 +93,13 @@ module.exports = ({ prisonApi, whereaboutsApi, oauthApi, logError }) => async (r
...
           {
-            html: `<a href="${offenderUrl}" class="govuk-link">${offenderName}</a>`,
+            html: `<a href="${offenderUrl}" class="govuk-link">${offenderName} - ${offenderNo}</a>`,
             attributes: {
               'data-sort-value': appointment.lastName,
             },
           },
           {
-            text: offenderNo,
+            text: prisonerDetails.assignedLivingUnit?.description,
```
Comment (JamieDraperUK) [inline backend/routes/appointments/viewAppointmentsRouter.js:102]: :heart:

Comment (JamieDraperUK) [inline backend/routes/appointments/viewAppointmentsRouter.js:102]: Looks like prisonerDetails can be null too.

File: backend/routes/appointments/viewAppointmentsRouter.js [hunk 2]
```diff
@@ -57,6 +57,11 @@ module.exports = ({ prisonApi, whereaboutsApi, oauthApi, logError }) => async (r
...
           }))
 
+        const prisonerDetails = await prisonApi.getDetails(res.locals, offenderNo, true).catch(error => {
+          logError(req.originalUrl, error, serviceUnavailableMessage)
+          return null
+        })
+
         const createdBy =
           videoLinkLocation &&
           videoLinkLocation.createdByUsername &&
```
Comment (JamieDraperUK) [inline backend/routes/appointments/viewAppointmentsRouter.js:67]: Could use the fancy optional thing here too.

File: backend/tests/viewAppointmentsController.test.js [hunk 3]
```diff
@@ -202,6 +204,12 @@ describe('View appointments', () => {
           lastName: 'THREE',
         })
 
+      prisonApi.getDetails
+        .mockResolvedValueOnce({ assignedLivingUnit: { description: '1-1-1' } })
+        .mockResolvedValueOnce({ assignedLivingUnit: { description: '2-1-1' } })
+        .mockResolvedValueOnce({ assignedLivingUnit: { description: '3-1-1' } })
+        .mockResolvedValueOnce({})
```
Comment (jsrobertson) [inline backend/tests/viewAppointmentsController.test.js:211]: Don't need this. Removing.

**Review by JamieDraperUK — APPROVED** [review]
Nice one :+1:

**Pull request merged**